### DISCUSSION TITLE
New (Wannabe) User

### DISCUSSION THREAD

**Discussion by johnstonesnow** [evilsocket/opensnitch#1122] [Q&A]
I am leaving Apple. One thing I will greatly miss is LittleSnitch. I am moving to Fedora. 

Is it possible to run this wonderful software on Fedora? I can't find it in the Fedora repo/software app

Thanks

**Comment by gustavo-iniguez-goya** [comment]
Sure @johnstonesnow !

Firstly download the packages from the release section: https://github.com/evilsocket/opensnitch/releases/tag/v1.6.5

1) https://github.com/evilsocket/opensnitch/releases/download/v1.6.5/opensnitch-1.6.5-1.x86_64.rpm
2) https://github.com/evilsocket/opensnitch/releases/download/v1.6.5/opensnitch-ui-1.6.5-1.noarch.rpm

And install them from a terminal (it's not really super necessary, but it helps to see the erorrs if it fails. It shoudn't..): https://github.com/evilsocket/opensnitch?tab=readme-ov-file#rpm

Basically: `$ sudo yum localinstall opensnitch-1*.rpm; sudo yum localinstall opensnitch-ui*.rpm`

Alternatively, there're unofficial packages maintained by some users: https://software.opensuse.org/package/opensnitch

**Reply by johnstonesnow** [reply]
Thanks very much!
I should have said, I can barely use github, never mind Terminal :) 
So do I just click to download the two files in 1 and 2 above, then run the Terminal command and that's it? Wow! :) 
I have never downloaded any software outside of the Software app/repo for Fedora, I am wary. I take it this is 'safe' to download and use, even though it's not in the fedora software repository?
Thanks again, I can't wait to spread the word about this, I have a ton of friends all giving up on Apple and all of them used LS as I did for many years. SO nice to have similar options to block spyware and unwanted big tech connections by installed apps (and maybe even OS)

**Reply by gustavo-iniguez-goya** [reply]
Yeah, just double click on both files, and it should be installed (crossfinger...). 

>  I take it this is 'safe' to download and use, even though it's not in the fedora software repository?

well, as far as I can tell, yes. Of course, it'd be better if a Suse or Fedora developer packaged it for their distro.

Bear in mind that opensnitch is not LS, it's similar but it works differently in many aspects.. hope you like it!

**Reply by johnstonesnow** [reply]
Thanks, I am not keen on LittleSnitch (the company) to be honest, but I like what the software DOES. If OS 'does' the same thing (blocks telemetry and other nasties) then I am sure I'll love it! 

When you say "just double click on both files, and it should be installed" - am I right in thinking you mean download them both AND run the install command? Or just click to download and that's them installed? I don't think so, but I have very little idea about Linux so wanted to confirm either way. thanks again. (I'm also gonna ask Fedora devs if someone could consider packaging it, I'm sure my request will get support)

**Reply by gustavo-iniguez-goya** [reply]
I just tested it on Fedora 40: double clicking on the download .rpm files works ok (on some distros used to fail).

But you'll need to install an extension to display the icon in the systray and access the GUI easily: https://github.com/evilsocket/opensnitch/wiki/GUI-known-problems#opensnitch-icon-does-not-show-up-on-gnome-shell

```bash
~ $ sudo yum install gnome-shell-extension-appindicator.noarch
```
LOGOUT from the user session.
```bash
~ $ gnome-extensions enable appindicatorsupport@rgcjonas.gmail.com
~ $ killall opensnitch-ui
```
Launch the GUI again.

**Reply by johnstonesnow** [reply]
You're a diamond, thank you. I am away at the mo but once back at desk I look forward to going through this. Thanks for providing this program, I'd feel naked without some kind of connection monitoring after all these years
